charts/monasca/templates/role-binding.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.rbac.enabled }}
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: "monasca-role-binding"
```

Comment:
Ahh, that was supposed to be templated but I removed it while debugging. Seems I was a bit too eager to post the PR!